Services/src/main/resources/webutils/templates/webutils-templates.html

```diff
@@ -931,6 +931,10 @@ <h4 class="modal-title">${attr('title')}</h4>
 									<input type="text" name="${fieldDef.name}" placeholder="${fieldDef.label}" 
 										ng-model="${modelField}" yk-read-only="${fieldDef.readOnly}" ng-change="${validateFunc}"></input>
 								</if>
+								<else-if test="context.fieldDef.fieldType == 'COLOR'">
+									<yk-color-field name="${fieldDef.name}" label="color" ng-model="${modelField}" placeholder="${fieldDef.label}"

```

Comment:
Label should not be hard coded as "color" here. You should get it from modelDef and use it.